modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,132 @@
+| [Subchannel Module](https://subchannel-dev.hpc.inl.gov/site/index.html) | Subchannel Scale                       | Incompressible or Weakly-Compressible \\ \\ Single-Phase \\ \\ Single- or Multi-Component Flow                                                               | Typically, 3D \\ \\ Can be used in 1D and 2D                       | 100,000               | 10 seconds      | Flow development through nuclear reactor fuel assembly \\ \\ Thermal-hydraulics analysis of nuclear reactor assembly blockage \\ \\ Natural convection cooling in nuclear reactors low-flow assemblies |
+| [Thermal-Hydraulics Module](modules/thermal_hydraulics/index.md)        | Lumped-Parameters Simulations          | Compressible \\ \\ Single-Phase; Single-Component Flow                                                                                                       | 1D, 0D                                                             | 100                   | 10 seconds      | Heat extraction unit from nuclear reactor core \\ \\ Thermal loops with significant compressibility effects                                                                                            |
+| [Porous Flow Module](modules/porous_flow/index.md)                      | Coarse-Mesh CFD                        | Incompressible, Weakly-Compressible, or Fully-Compressible Porous Flow (no inertial term) \\ \\ Single- or Multi-Phase \\ \\ Single- or Multi-Component Flow | Typically, 2D, 2D axisymmetric, or 3D \\ \\ Can also be used in 1D | 10,000                | 1 minute        | Flow through fractured porous media \\ \\ Underground coal mining \\ \\ CO storage in saline aquifers                                                                                                  |
+| [Scalar Kernels System](syntax/ScalarKernels/index.md)                  | 0D ODE                                 | Incompressible                                                                                                                                               | 0D                                                                 | 10                    | 1 second        | Bernoulli-like formulations for pressure drop                                                                                                                                                          |
+
+## Open-Source Applications for Advanced Engineering Modeling
+
```

Comment:
```suggestion
## Open-Source MOOSE-based Wrappers for External Fluid Dynamics Applications

```

Reply:
How about this change? I agree that Cardinal is a very useful interface between Nek and everything MOOSE-based. But Nek is not MOOSE-based, and if I understand @tanoret 's proposal we are trying to advertise the tools we are working on a website we have more control over (compared to the NEAMS website). Adding advanced engineering modeling might make some visitors think that the tools mentioned before are not advanced and not for engineering.

Reply:
I combined all into one table now, so this suggestion probably doesn't apply anymore.